js/goto.js

```diff
@@ -17,7 +17,7 @@ function redirectionHandler(data) {
 }
 
 function showSignupFormIfIncomplete() {
-  fetch('https://api.realdevsquad.com/users/self', {
+  fetch('http://api.realdevsquad.com/users/self', {
```

Comment:
Why using `http` ?

Reply:
Missed it. Very good catch 👁 @sahsisunny